tests/roles/mariadb_copy/tasks/main.yaml

```diff
@@ -1,8 +1,8 @@
-- name: get podified MariaDB IP
+- name: get podified MariaDB service cluster IP
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc get -o yaml pod mariadb-openstack | grep podIP: | awk '{ print $2; }'
+    oc get svc --selector "cr=mariadb-openstack" -ojsonpath='{.items[0].spec.clusterIP}'
```

Comment:
Won't this break if we have 2 MariaDB instances? (Wouldn't cell 1 instance also match the CR selector?)

+1 on the clusterIP but i think we should keep selecting by pod name?

Reply:
I checked on my env, it works OK for main and cell db svc

Reply:
:+1: thanks